homeassistant/bridge.py

```
@@ -0,0 +1,298 @@
+                            "service": "turn_off",
+                            "service_data": {"entity_id": "input_boolean.test_switch"}
+                        }))
+                        self.message_id += 1
+                    else:
+                        logger.error("WebSocket no está conectado, usando API REST como fallback")
+                        await self.call_ha_service("input_boolean.turn_off", {"entity_id": "input_boolean.test_switch"})
+                    logger.info("=====================================")
+                    logger.info("¡APAGANDO input_boolean.test_switch POR DEFECTO!")
+                    logger.info("=====================================")
+
+    async def connect_to_ha(self):
```

Ruff (0.8.2)

Redefinition of unused `connect_to_ha` from line 55

(F811)